editoast/src/views/timetable/stdcm.rs

```diff
@@ -100,6 +99,8 @@ enum StdcmError {
     TrainSimulationFail,
     #[error("Path items are invalid")]
     InvalidPathItems { items: Vec<InvalidPathItem> },
+    #[error("Invalid request: {message}")]
+    InvalidRequest { message: String },
```

Comment:
This is a bit vague imo, maybe `InvalidConsistLength` and `InvalidConsistMass` with numeric fields would be more descriptive?